src/core/include/units/magnitude.h

```diff
@@ -241,6 +357,14 @@ static constexpr bool is_magnitude<magnitude<BPs...>> = true;
 template<typename T>
 concept Magnitude = detail::is_magnitude<T>;
 
+/**
+ * @brief  Free-function access to the value of a Magnitude in a desired type.
+ *
+ * Can avoid the need for an unsightly `.template` keyword.
+ */
+template<typename T>
+T get_value(Magnitude auto m) { return decltype(m)::template value<T>; }
```

Comment:
Hah, this is exactly what I meant above.

Reply:
Actually probably this function is enough and we do not need to duplicate instances here?